doc/Reference/Reactive/state.md

```diff
@@ -118,6 +174,12 @@ public async ValueTask SetCurrent(CancellationToken ct)
 
 For value types and strings, you also have a `Set` **which does not ensure the respect of the ACID properties**.
 
+```csharp
+public static Task Set<T>(this IState<T> state, T value, CancellationToken ct = default);
+```
```

Comment:
Looking [here](https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive/Core/State.Extensions.cs), it seems `Set` is deprecated. We should be using `SetAsync` instead.

Reply:
And edit that current example code that is still using Set